bin/titan/main.go

```diff
@@ -23,16 +23,19 @@ import (
 	"github.com/distributedio/titan/db"
 	"github.com/distributedio/titan/metrics"
 	"github.com/distributedio/titan/server"
+	"github.com/distributedio/titan/tools/dump"
 )
 
 func main() {
 	var showVersion bool
 	var confPath string
 	var pdAddrs string
+	var dumpRedisCommand string
 
 	flag.BoolVar(&showVersion, "v", false, "Show Version")
 	flag.StringVar(&confPath, "c", "conf/titan.toml", "conf file path")
 	flag.StringVar(&pdAddrs, "pd-addrs", "", "pd cluster addresses")
+	flag.StringVar(&dumpRedisCommand, "dump", "", "dump redis command and duration to file")
```

Comment:
maybe `bool` is better

Reply:
the `dumpRedisCommand` is a param what dump's command save directory